src/main/java/io/temporal/samples/complex/ComplexWorkflowImpl.java

```diff
@@ -0,0 +1,8 @@
+package io.temporal.samples.complex;
+
+public class ComplexWorkflowImpl implements ComplexWorkflow {
+  @Override
+  public void handleLambda(Input input) {
```

Comment:
real logic is here. Arbitrary workload should be done in Activity.  WorkflowMethod only contains deterministic code.